### DISCUSSION TITLE
How to switch account?

### DISCUSSION THREAD

**Discussion by binary4cat** [zbirenbaum/copilot.lua#122] [Q&A] [answered]
I want to log out of the current account, and then log in to another account, but I find that there is no option to log out, what should I do to switch accounts?

**Comment by MunifTanjim** [comment, marked as answer]
Check: https://github.com/zbirenbaum/copilot.lua/discussions/120#discussioncomment-4929370

You can run `:Copilot auth signout` to sign out of current account.

The auth information is usually stored in `~/.config/github-copilot/hosts.json` file.

**Reply by binary4cat** [reply]
Thank you very much for your answer.
